### PR TITLE
DQI changes in attribute grid, labels and options

### PR DESCRIPTION
**Pull request by fleguellec**
Changes to reflect new features on DQI: spell-check on attributes and attribute options - missing some screenshots and gif

### CODE REVIEW

File: content/md/products-super-power/improve-data-quality.md [hunk 1]
```diff
@@ -69,16 +69,18 @@ To help you even further, in the `Attributes` panel, you can select spelling cor
...
 Title formatting suggestion is available for English locales.
-:::
 
-::: info
 Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
 :::
 
+From the `data quality panel`, you can view attribute labels and options with spelling mistakes. By clicking on the link, you are led to the **settings** screen from which you can correct these mistakes hence improving your consistency grade.
+
+![Image data quality spellcheck on attribute](../img/data-quality-attribue-spelling.gif)
```
Comment (olivierpontier) [inline content/md/products-super-power/improve-data-quality.md:81]: Is it a new image? I don't see it in the list of files joined to this pull request.

**Comment by olivierpontier** [comment]
The picture `content/md/what-is/img/data-quality-panel copy.png` contains a space character in the name (not very url friendly) and it seems to not be used in documentation.

File: content/md/products-super-power/improve-data-quality.md [hunk 2]
```diff
@@ -69,16 +69,18 @@ To help you even further, in the `Attributes` panel, you can select spelling cor
 ::: info
 Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, Swedish, Norwegian and Portuguese (Brazilian) locales.
 Spell check is usable with localizable text and text area attributes.
-:::
+If you use a single locale, then spell-check are usable on text and text area attributes.
```
Comment (camillebourjade) [inline content/md/products-super-power/improve-data-quality.md:72]:
```suggestion
If you use a single locale, then spell check is usable on text and text area attributes.
```

File: content/md/products-super-power/improve-data-quality.md [hunk 3]
```diff
@@ -69,16 +69,18 @@ To help you even further, in the `Attributes` panel, you can select spelling cor
...
 
-::: info
 Title formatting suggestion is available for English locales.
-:::
 
-::: info
 Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
 :::
 
+From the `data quality panel`, you can view attribute labels and options with spelling mistakes. By clicking on the link, you are led to the **settings** screen from which you can correct these mistakes hence improving your consistency grade.
```
Comment (camillebourjade) [inline content/md/products-super-power/improve-data-quality.md:79]:
```suggestion
From the `data quality panel`, you can view attribute labels and options with spelling mistakes. By clicking on the link, you land on the **settings** screen from which you can correct these mistakes hence improving your consistency grade.
```

File: content/md/what-is/understand-data-quality.md [hunk 4]
```diff
@@ -29,14 +29,15 @@ From the `data quality insights` panel, by clicking on the "Display all missing
 # How is consistency calculated?
 
 The **consistency grade** is calculated on several criteria. The mix of the following criteria helps the PIM determine the grade for consistency:
-- Spell check: localizable text and text areas are checked for spelling mistakes.
+- Spell check: localizable text and text areas, attribute labels and attribute options are checked for spelling mistakes.
```
Comment (camillebourjade) [inline content/md/what-is/understand-data-quality.md:32]: consistency: you sometimes write spell check and sometimes spell-check. I recommend the first

File: content/md/what-is/understand-data-quality.md [hunk 5]
```diff
@@ -29,14 +29,15 @@ From the `data quality insights` panel, by clicking on the "Display all missing
...
 - Title formatting: the attribute used as **main label** is checked to optimize writing.
 - Uppercases formatting: the PIM checks whether text area attributes are correctly formatted for upper cases.
 - Lowercases formatting: the PIM checks whether text area attributes are correctly formatted for lower cases.
 
 ::: info
 Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, Swedish, Norwegian and Portuguese (Brazilian) locales.
 Spell check is usable with localizable text and text area attributes.
+If you use a single locale, then spell-check are usable on text and text area attributes.
```
Comment (camillebourjade) [inline content/md/what-is/understand-data-quality.md:40]:
```suggestion
If you use a single locale, then spell check is usable on text and text area attributes.
```

File: content/md/what-is/understand-data-quality.md (hunk 4 above)
Comment (fleguellec) [reply content/md/what-is/understand-data-quality.md:32]: thanks!

File: content/md/products-super-power/improve-data-quality.md (hunk 1 above)
Comment (fleguellec) [reply content/md/products-super-power/improve-data-quality.md:81]: no, it's not a new image.

**Pull request merged**